config.yaml

```diff
@@ -42,34 +42,18 @@ build_iso:
     packages_dir: "result/packages/latest"
```

Comment:
Actually, this is a CentOS group that was modified by us, it will still exist.
Clarifying this is commit message.